### ISSUE TITLE
[lua-nginx-module] Nginx 1.11.11 Compilation Error

### ISSUE THREAD

**Issue by kazuhisya** [kazuhisya/nginx-module-ndk-lua#2]
SeeAlso: lua-nginx-module #1016

```
lua-nginx-module-0.10.7/src/ngx_http_lua_headers.c: In function 'ngx_http_lua_ngx_req_raw_header':
lua-nginx-module-0.10.7/src/ngx_http_lua_headers.c:151:15: error: incompatible types when assigning to type 'struct ngx_buf_t *' from type 'ngx_chain_t'
             b = hc->busy[i];
               ^
lua-nginx-module-0.10.7/src/ngx_http_lua_headers.c:227:15: error: incompatible types when assigning to type 'struct ngx_buf_t *' from type 'ngx_chain_t'
             b = hc->busy[i];
               ^
make[1]: *** [objs/addon/src/ngx_http_lua_headers.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory `/root/rpmbuild/BUILD/nginx-1.11.11'
make: *** [modules] Error 2


RPM build errors:
error: Bad exit status from /var/tmp/rpm-tmp.3VLAab (%build)
    Bad exit status from /var/tmp/rpm-tmp.3VLAab (%build)
```

**Comment by blackhu** [comment]
I also encountered the same problem。
./configure --prefix=/usr/local/nginx --with-ld-opt="-Wl,-rpath,/usr/local/luajit-2.0.4/lib" --with-pcre=/root/pcre-8.40  --add-module=/root/lua-nginx-module-0.10.7 --add-module=/root/ngx_devel_kit-0.3.0

**Comment by kazuhisya** [comment]
Temporarily fixed by workaround.
https://github.com/kazuhisya/nginx-module-ndk-lua/commit/b933c9cd2d684f757929d1127791911e347e1707#diff-3118146bf3c05be55c3fc16429190597

See Also:
https://github.com/openresty/lua-nginx-module/pull/1017/commits
https://github.com/defanator/lua-nginx-module/compare/0064324...93f8e8.diff

This workaround(`defanator_lua-nginx-module_compare_0064324-93f8e8.patch`) is scheduled to be deleted after updating the upstream.

**Comment by kazuhisya** [comment]
Fixed: be13b14
